docs/model_pipeline.rst

```diff
@@ -70,13 +72,23 @@ Rules
 
    prepare_resources/*
 
+
 Outputs
 -------
 
 Output files are saved in :file:`results/_resources`.
 
 The resources are preprocessed data that serve as material for building scenarios.
-Because they are a first intermediate result, resources will be saved in :file:`results/_resources`.
+They are a first intermediate result in oemof-B3.
+
+File storage
```

Comment:
Do we really need these headings (file storage, schema, conventions)? They do not appear in the other sections...

Reply:
No we don't really need them, however I found that they improved the overview. But we can also stay consistent with the other sections. Should I revert the changes @jnnr?

Reply:
As agreed, I have pulled up the Schema and Conventions sections to Raw Data with commit https://github.com/rl-institut/oemof-B3/pull/280/commits/dd9d7487308e0ead4518342d310cbec0e9440127.